instance/models/openedx_appserver.py

```diff
@@ -189,6 +189,13 @@ class Meta:
             "becomes unavailable."
         )
     )
+    celery_broker_transport = models.CharField(
+        max_length=16,
+        blank=False,
+        null=False,
+        default="redis",
+        help_text='The Celery broker transport to use.  Supported values include "redis" and "amqp".',
```

Comment:
We already have validation once we create an appserver – the function building the Celery broker config will raise an exception if this isn't one of the two currently supported values.

I didn't want to add all the usual constants, since this felt to verbose, but just adding the `choices` argument here certainly doesn't hurt.

Reply:
Thanks!